user-journey-tests/config/browserstackMetrics.js

```diff
@@ -0,0 +1,41 @@
+const defaultMinimum = 6
+
+export const getBrowserStackRunners = async () => {
+  const username = process.env.BROWSERSTACK_USER
+  const accessKey = process.env.BROWSERSTACK_KEY
+
+  try {
+    const response = await fetch(
+      'https://api.browserstack.com/automate/plan.json',
+      {
+        headers: {
+          Authorization: 'Basic ' + btoa(`${username}:${accessKey}`)
+        }
+      }
+    )
+
+    if (!response.ok) {
+      throw new Error(`HTTP error! status: ${response.status}`)
+    }
+
+    const data = await response.json()
+    const {
+      parallel_sessions_max_allowed: threadsAllowed,
+      parallel_sessions_running: threadsRunning
+    } = data
+    const availableThreads = threadsAllowed - threadsRunning
+    const threadsToUse = Math.floor(availableThreads / 2)
+
+    if (threadsToUse < defaultMinimum && threadsToUse < availableThreads) {
```

Comment:
I'm not sure if it's just my eyesight, but this doesn't read quite right to me. 

Threads to use will always be < available threads, because it's calculated as `const threadsToUse = Math.floor(availableThreads / 2)`

It looks to me that we want: 

```
if (defaultMinimum > availableThreads) {
  return availableThreads
} else {
  return defaultMinimum
}
```

Also, isn't it more of a defaultMaximum?

Reply:
can you tell im working on 3hrs sleep?